### DISCUSSION TITLE
NonceMiner

### DISCUSSION THREAD

**Discussion by gavinowens** [revoxhere/duino-coin#1534] [Q&A]
I was just reading about NonceMiner, and it sounds really cool. Why did you ban it? I know this is a centralized project, but isn't competitive mining in everyone's best interest? The program looks like it can still be run on things like a Raspberry Pi, so it wouldn't prevent small devices from mining. Why was it banned?

**Comment by GerardoNevarez** [comment]
Read the whitepaper behind Duino Coin.

**Reply by gavinowens** [reply]
I have, I still don't understand.
